blazor/rating/templates.md

```diff
@@ -0,0 +1,260 @@
+        font-weight: normal;
+        font-style: normal;
+    }
+
+    [class^="sf-rating-"], [class*=" sf-rating-"] {
+        font-family: 'rating' !important;
+        speak: none;
+        font-style: normal;
+        font-weight: normal;
+        font-variant: normal;
+        text-transform: none;
+        line-height: 1;
+        -webkit-font-smoothing: antialiased;
+        -moz-osx-font-smoothing: grayscale;
+    }
+
+    .sf-rating-heart:before {
+        content: "\e702";
+    }
+</style>
+
+```
+
+![Blazor Rating Component with Empty Template](./images/blazor-rating-empty-template.png)
+
+## Full Template
+
+You can access information about the rated item by setting [FullTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_FullTemplate) property.
+
+Below example demonstrates the Full Template of Rating.
+
+```cshtml
+
+@using Syncfusion.Blazor.Inputs
+
+<SfRating>
+    <FullTemplate>
+        <span class='custom-font sf-icon-star'></span>
+    </FullTemplate>
+    <EmptyTemplate>
+        <span class='custom-font sf-icon-star'></span>
+    </EmptyTemplate>
+</SfRating>
```

Comment:
If you are using the same content for full and empty templates what's the need for 2 templates?

Reply:
I have updated it as per comments.